fw/ss_skb.h

```diff
@@ -166,6 +166,21 @@ ss_skb_queue_split(struct sk_buff *skb_head, struct sk_buff *skb)
 	skb_head->prev = prev;
 }
 
+/*
+ * Insert @nskb in the list after @skb. Note that standard
+ * kernel 'skb_insert()' function does not suit here, as it
+ * works with 'sk_buff_head' structure with additional fields
+ * @qlen and @lock; we don't need these fields for our skb
+ * list, so a custom function had been introduced.
+ */
+static inline void
+skb_insert_after(struct sk_buff *skb, struct sk_buff *nskb)
```

Comment:
```suggestion
ss_skb_insert_after(struct sk_buff *skb, struct sk_buff *nskb)
```
just to make the name consistent with the other functions in the file